examples/python/dynamodb_logstore.py

```diff
@@ -43,14 +43,17 @@
 export DELTA_CONCURRENT_READERS=2
 export DELTA_TABLE_PATH=s3a://test-bucket/delta-test/
 export DELTA_DYNAMO_TABLE=delta_log_test
-export DELTA_STORAGE=io.delta.storage.DynamoDBLogStore
+export DELTA_STORAGE=io.delta.storage.DynamoDBLogStoreScala # TODO: remove `Scala` when Java version finished
 export DELTA_NUM_ROWS=16
+
+TODO: update this comment with proper delta-storage artifact ID (i.e. no _2.12 scala version)
+
 ./run-integration-tests.py \
   --test dynamodb_logstore.py \
   --python-only \
   --conf spark.jars.ivySettings=/workspace/ivy.settings \
          spark.driver.extraJavaOptions=-Dlog4j.configuration=file:debug/log4j.properties \
-  --packages io.delta:delta-contribs_2.12:${VERSION},org.apache.hadoop:hadoop-aws:3.3.1,com.amazonaws:aws-java-sdk-bundle:1.12.142
+  --packages io.delta:delta-storage_2.12:${VERSION},org.apache.hadoop:hadoop-aws:3.3.1,com.amazonaws:aws-java-sdk-bundle:1.12.142
```

Comment:
Should this be `delta-storage-dynamodb`?

Reply:
yup good catch, thanks